simulator/util/dummy_perception_publisher/README.md

```diff
@@ -0,0 +1,45 @@
+# dummy_perception_publisher
+
+## Purpose
+
+This node publishes the result of the dummy detection with the type of perception. f `real_use_param_` is True, the data containing the Feature is delivered, and if False, the data without the Feature is delivered.
```

Comment:
```suggestion
This node publishes the result of the dummy detection with the type of perception. If `real_use_param_` is True, the data containing the Feature is delivered, and if False, the data without the Feature is delivered.
```
Missing `I`. Please check carefully.

Reply:
f25d3be

Reply:
Sorry again, I double checked it.f25d3bea3fbbbb78cddb075db6344377a04566ec